src/screens/StudyScreen/WordCard.jsx

```diff
@@ -10,13 +10,17 @@ import { constructWordExample } from 'shared/utils/highlight';
 import { toCapitalize } from 'shared/utils/stringHelpers';
 import { getTextSize } from 'shared/utils/style';
 
-const DisplayText = ({ children, size, shrink, fontStyle }) => (
-  <Text size={size} flexShrink={shrink} fontFamily={fontStyle.toLowerCase()} _light={{ color: 'vhlight.50' }} _dark={{ color: 'vhdark.50' }}>
-    {children}
-  </Text>
-);
+const DisplayText = ({ children, size, shrink, fontStyle, isBilingual }) => {
```

Comment:
`DisplayText` is responsible for rendering text only. It should not get involved in bilingual logic or not. Move this logic to the parent component.